docs/getting-started/configure-app-with-plugins.md

```diff
@@ -22,63 +22,56 @@ The following steps assume that you have
+   [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/). Frontend UI Plugins are generally added to the `app` folder, while Backend Plugins are added to the `backend` folder. In the example above, the plugin is added to the `app` package because we are adding the frontend plugin.
 
-2. Add the `EntityCircleCIContent` extension to the entity pages in the app:
+2. Now, modify your app routes to include the Router component exported from the tech radar, for example:
 
-   ```tsx title="packages/app/src/components/catalog/EntityPage.tsx"
+   ```tsx title="packages/app/src/App.tsx"
    /* highlight-add-start */
-   import {
-     EntityCircleCIContent,
-     isCircleCIAvailable,
-   } from '@circleci/backstage-plugin';
+   import { TechRadarPage } from '@backstage-community/plugin-tech-radar';
    /* highlight-add-end */
 
-   const cicdContent = (
-     <EntitySwitch>
-       {/* ... */}
-       {/* highlight-add-next-line */}
-       <EntitySwitch.Case if={isCircleCIAvailable}>
-         <EntityCircleCIContent />
-       </EntitySwitch.Case>
-       ;{/* highlight-add-end */}
-     </EntitySwitch>
+   const routes = (
+     <FlatRoutes>
+       /* highlight-add-start */
+       <Route path="/tech-radar" element={<TechRadarPage />} />
+       /* highlight-add-end */
+     </FlatRoutes>
    );
    ```
 
-   This is just one example, but each Backstage instance may integrate content or
+   This is just one example, and if you'd like to continue adding the Tech Radar plugin you can do so by going [here](https://github.com/backstage/community-plugins/tree/main/workspaces/tech-radar/plugins/tech-radar), keep in mind each Backstage instance may integrate content or
    cards to suit their needs on different pages, tabs, etc. In addition, while some
-   plugins such as this example are designed to annotate or support specific software
-   catalog entities, others may be intended to be used in a stand-alone fashion and
-   would be added outside the `EntityPage`, such as being added to the main navigation.
+   plugins such as this example are designed to be used in a stand-alone fashion,
+   others may be intended to annotate or support specific software catalog entities
+   and would be added elsewhere in the app.
 
 3. _[Optional]_ Add a proxy config:
 
-   Plugins that collect data off of external services may require the use of a proxy service.
-   This plugin accesses the CircleCI REST API, and thus requires a proxy definition.
+   Plugins that collect data off of external services may require the use of a proxy service, to support this you can add the following settings to your `app-config.yaml` file:
 
    ```yaml title="app-config.yaml"
    proxy:
-     '/circleci/api':
-       target: https://circleci.com/api/v1.1
+     '/<proxy-path>/api':
+       target: https://<your-target-instance>.com
        headers:
-         Circle-Token: ${CIRCLECI_AUTH_TOKEN}
+         Authorization: ${YOUR_AUTH_TOKEN}
```

Comment:
perhaps we can kill this section now? 🤔

Reply:
Yeah I'm also thinking it's not really necessary here anymore! 🫡

Reply:
Done [15d2a6f](https://github.com/backstage/backstage/pull/29316/commits/15d2a6fd54a057df95cd8e18c5e8dfcb87b8c083)